cmd/deploy/deploy.go

```diff
@@ -150,7 +150,9 @@ func Deploy(ctx context.Context) *cobra.Command {
 					if err != nil {
 						return err
 					}
-					nsCmd.Create(ctx, &namespace.CreateOptions{Namespace: okteto.Context().Namespace})
+					if err := nsCmd.Create(ctx, &namespace.CreateOptions{Namespace: okteto.Context().Namespace}); err != nil {
```

Comment:
We should be returning an error here, we will fail later on because the ns doesn't exist